test/from-draft7/to-2019-09/dependencies-to-dependentRequired.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "title": "`dependencies` has properties whose subschema has type array",
+        "from": {
+            "dependencies": {
+                "foo": [ "bar", "baz" ]
+            }
+        },
+        "to": {
+            "dependentRequired": {
+                "foo": [ "bar", "baz" ]
+            }
+        }
+    },
+    {
+        "title": "`dependencies` has properties whose subschema does not have type array",
+        "from": {
+            "dependencies": {
+                "foo": {
+                    "minProperties": 2
+                }
+            }
+        },
+        "to": {
+            "dependencies": {
```

Comment:
Shouldn't this be getting transformed to `dependentSchemas`

Reply:
@jviotti I put that test to show that it shouldn't be transformed in this case, when schema doesn't match

Reply:
Not sure I'm following. In Draft 7, `dependencies` can take either an array of properties of schemas. If a property is a schema, then on 2019-09 it gets transformed to `dependentSchemas`, right?